deploy/crds/ssp.kubevirt.io_v1_ssp_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: ssp.kubevirt.io/v1
+kind: SSP
+metadata:
+  name: example-ssp
+spec: {}
```

Comment:
Can you add new line at the bottom?

Reply:
Added. I forgot to configure IDE to add them.